docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -95,45 +78,45 @@ The main features of the Autofill Manager API include:
    }
    ```
 
-   You can use the autofill manager functions after receiving `AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED` in `manager_connection_status_changed_cb()`.
+   You can use the Autofill manager functions after receiving `AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED` in `manager_connection_status_changed_cb()`.
 
-## Getting installed autofill service list
+## Get Installed Autofill Service List
 
 To get the installed autofill service list:
 
 1. To access Autofill service information, create a callback function using the `autofill_service_info_cb()` function, which is invoked for each installed Autofill service application:
 
-```c
-#include <app_manager.h>
+   ```c
+   #include <app_manager.h>
 
-static bool autofill_service_info_cb(const char* app_id, void* user_data)
-{
-    app_info_h app_h;
-    char *label = NULL;
+   static bool autofill_service_info_cb(const char* app_id, void* user_data)
+   {
+       app_info_h app_h;
+       char *label = NULL;
 
-    app_info_create(app_id, &app_h);
-    app_info_get_label(app_h, &label);
+       app_info_create(app_id, &app_h);
+       app_info_get_label(app_h, &label);
 
-    dlog_print(DLOG_INFO, LOG_TAG, "app id : %s, label : %s", app_id, label);
+       dlog_print(DLOG_INFO, LOG_TAG, "app id : %s, label : %s", app_id, label);
 
-    if (label)
-        free(label);
+       if (label)
+           free(label);
 
-    app_info_destroy(app_h);
+       app_info_destroy(app_h);
 
-    return true;
-}
-```
+       return true;
+   }
+   ```
 
 2. To get the installed Autofill service by invoking a callback function for each installed Autofill service application, use the `autofill_manager_foreach_autofill_service()` function:
```

Comment:
Get the installed autofill service by invoking a callback function for each installed autofill service with the `autofill_manager_foreach_autofill_service()` function:

Reply:
fixed